### DISCUSSION TITLE
How to fix `auto_lr_find`?

### DISCUSSION THREAD

**Discussion by eigenvivek** [Lightning-AI/pytorch-lightning#16633] [Lightning App API: LightningApp, LightningFlow, LightningWork]
I am trying to use the tuner in `Trainer` to optimize my learning rate, but keep running into a divergence loss (see the error below).

My training and validation sets both have 5000 samples each, so I don' the size of the dataloader is the problem. I've increased the width of the learning rates to search over, but that doesn't fix the issue.

```
LR finder stopped early after 2 steps due to diverging loss.
Failed to compute suggestion for learning rate because there are not enough points. Increase the loop iteration limits or the size of your dataset/dataloader.
```
